src/odemis/acq/fastem.py

```diff
@@ -27,6 +27,9 @@
 import time
 
 
+SINGLE_BEAM_ROTATION_DEFAULT = 0  # 0 is typically a good guess (better than multibeam rotation of ~0.0157 rad)
+MULTI_BEAM_ROTATION_DEFAULT = 0.015707963  # 0.9 degrees
```

Comment:
Is there any chance that this value is generic? If any system can have a sort of "random" number, then I'd suggest to just default to 0 for all the rotations.